### DISCUSSION TITLE
How to change the admin password?

### DISCUSSION THREAD

**Discussion by buhtigpater** [ckulka/baikal-docker#82] [Q&A] [answered]
Hello, everyone

I'm not sure if I'm in the right place with my question. I'm running Baikal in a Docker container. Now I want to change the admin password? How do I do that?

Thanks for helping me and others

**Comment by ckulka** [comment]
Hi @buhtigpater, a better place would be over at [sabre-io/Baikal](https://github.com/sabre-io/Baikal) since this is a question about how to do something in Baikal, not something related to the Docker images.

I found this here though in their issues: https://github.com/sabre-io/Baikal/issues/196#issuecomment-731747646

**Reply by buhtigpater** [reply]
Good evening ckulka

Thanks for your prompt response. Sadly it did not work. Regardless, I'll rebuild the server and then just populate the db.sqlite.

Thank you also for the container, you've already helped me a lot. - Keep it up!

**Comment by buhtigpater** [comment, marked as answer]
got it!

1. backup the existing config file

   ```sh
   cd /var/www/baikal/config
   cp -p baikal.yaml baikal.yaml_keep
   ```

1. generate a new password hash

   ```sh
   echo -n 'admin:BaikalDAV:MyNewTopSecretPassword' | sha256sum | cut -d' ' -f1`
   # new password hash: c06c5eedc0950a347fdffc7270605db834a4d57ece3cf88bb2924723f1edc900
   ```

1. get old password hash

   ```sh
   grep admin_passwordhash baikal.yaml
   # admin_passwordhash: 41739d5e231d38110d4c5e8c33dd5b4d4f11246d8d02aafa78ce599455e15b0f
   ```

1. replace old hash with new hash

   ```sh
   sed 's/old password hash/new password hash/' baikal.yaml.keep > baikal.yaml`
   ```

**Reply by ckulka** [reply]
Thanks! I've updated the formatting and marked it as the answer for others who come across this in the future

**Reply by listenandresolve** [reply]
Thank you so much mate !
Just did it, you save my calendar.

**Reply by garhow** [reply]
Thank you so much!!
